src/endpoints/seed/index.ts

```diff
@@ -60,24 +59,6 @@ export const seed = async ({
 }): Promise<void> => {
   payload.logger.info('Seeding database...')
   if (!incremental) {
-    payload.logger.info(`— Clearing collections and globals...`)
```

Comment:
Why was this change made?

Reply:
We don't have any globals anymore since we changed the footer to be a collection for each tenant. Is my understanding of the change incorrect?